WordPress/WordPressTest/ContextManagerTests.swift

```diff
@@ -373,6 +373,51 @@ class ContextManagerTests: XCTestCase {
         try XCTAssertEqual(contextManager.mainContext.count(for: request), 1)
     }
 
+    func testUpdateUsingSyncAPI() throws {
```

Comment:
What do you think of using a name that tells the reader this is meant to document an issue? Or a comment?

Alternatively, we could move the final async part of the test in a dedicated test.

Reply:
I've added a comment in https://github.com/wordpress-mobile/WordPress-iOS/pull/20145/commits/ab703dac366fcc49cbdcf8a52e128bed7735a70e